components/submit-expense/PickCollectiveStep.tsx

```diff
@@ -2,20 +2,38 @@ import React from 'react';
 import { gql, useQuery } from '@apollo/client';
 import clsx from 'clsx';
 import { uniqBy } from 'lodash';
-import { FormattedMessage } from 'react-intl';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
 
 import { CollectiveType } from '../../lib/constants/collectives';
 import { API_V2_CONTEXT } from '../../lib/graphql/helpers';
-import { CollectiveOptionQuery, CollectiveOptionQueryVariables } from '../../lib/graphql/types/v2/graphql';
+import {
+  ChildAccountPickerQuery,
+  ChildAccountPickerQueryVariables,
+  CollectiveOptionPickerQuery,
+  CollectiveOptionPickerQueryVariables,
+  CollectiveOptionQuery,
+  CollectiveOptionQueryVariables,
+} from '../../lib/graphql/types/v2/graphql';
 
-import Avatar from '../Avatar';
+import { AvatarWithLink } from '../AvatarWithLink';
+import CollectivePicker from '../CollectivePicker';
 import CollectivePickerAsync from '../CollectivePickerAsync';
-import Loading from '../Loading';
-import { Button } from '../ui/Button';
+import LoadingPlaceholder from '../LoadingPlaceholder';
+import MessageBoxGraphqlError from '../MessageBoxGraphqlError';
 
 import { RadioCardButton } from './RadioCardButton';
 import { ExpenseForm } from './useExpenseForm';
 
+const I18nMessages = defineMessages({
+  CHILD_DESCRIPTION: {
+    defaultMessage:
+      '{childName} is an {parentName} {childType, select, EVENT {event} FUND {fund} PROJECT {project} other {account}}',
```

Comment:
This won't work well if `parentName` starts with a consonant, ie.:
> Test in an Babel event

I would suggest rewording to avoid the issue:
```suggestion
      '{childName} is {childType, select, EVENT {an event} FUND {a fund} PROJECT {a project} other {an account}} of {parentName}',
```